tests/core/snapd-refresh-vs-services-reboots/task.yaml

```diff
@@ -11,11 +11,13 @@ details: |
 # TODO: move this test to tests/regression/lp-1924805 ?
 
 systems:
-  - ubuntu-core-18-*
   - ubuntu-core-20-*
   # TODO: 2.49.2 does not work on UC22 because of ABI breakage and missing
   # libgcc_s.so.1
   # - ubuntu-core-22-*
+  # snapd 2.49.2 doesn't support snap-declaration format 5, which is used by the
+  # kernel on core18 systems
+  # - ubuntu-core-18-*
```

Comment:
Nitpick: this (along with its comment) could remain above the `# - ubuntu-core-20-*` line, so the test systems remain in order, whether they're commented out or not.

Reply:
Done!